### PR TITLE
Update the readme and contributing files

### LINKED ISSUE #515: Update README
Update README to describe relationship to learn-astropy repo and link to new contributing docs and what not.

### PR DESCRIPTION
**Pull request by adrn**
This updates / fixes the README and CONTRIBUTING files to be consistent with the new infrastructure. 

Closes #515

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Contributing
+
+We welcome new contributed Astropy tutorials and improvements to existing tutorials. Please see the [contributing guide](https://learn.astropy.org/contributing) for more information.
```
Comment (kelle) [inline CONTRIBUTING.md:3]:
```suggestion
We welcome both new tutorials covering astronomy-related concepts and improvements to existing tutorials. Please see the [contributing guide](https://learn.astropy.org/contributing) for more information on how to contribute.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+
+Contributing tutorial material
+------------------------------
+
+We are always interested in incorporating new tutorials into Learn Astropy and
+the Astropy Tutorials series. If you have Astropy or Astropy affiliated
+package-related tutorials that you would like to contribute to this repository,
+or if you have a separate tutorial series that you would like indexed by the
+Learn Astropy website, please see the [Contributing
```
Comment (kelle) [inline README.md:60]:
```suggestion
the Astropy Tutorials series. We welcome tutorials covering astro-relevant topics and they do not 
necessarily need to use the Astropy package in order to be hosted or indexed here.
If you have astronomy tutorials that you would like to contribute to this repository,
or if you have a separate tutorial series that you would like indexed by the
Learn Astropy website, please see the [Contributing
```

**Review by kelle — COMMENTED** [review]
some language tweaks.

**Comment by adrn** [comment]
Thanks!

**Pull request merged**